roles-and-groups.md

```diff
@@ -0,0 +1,153 @@
+- An example of a promotion can be found in the following [PR](https://github.com/hiero-ledger/governance/pull/176).
+- Once the PR is created, all the maintainers of the repo in question need to be added as reviewers and their vote should be provided.
+- The creator of the PR should notify all maintainers that a vote is now open.
+- Maintainers can vote in favor of the candidate's promotion by approving the PR with a comment indicating approval.
+- Maintainers can vote against the candidate's promotion by posting a comment in the PR along with their explanation.
+- Maintainers can abstain from voting by posting a comment in the PR along with their explanation.
+- The comment associated with a maintainers vote should be formatted as `[APPROVE|REJECT|ABSTAIN] - <explanation>`.
+- Maintainers are required to vote, if a maintainer does not participate in five consecutive votes they will be considered "inactive" which can be grounds for removal of maintainer status.
+- Once the PR has gathered the majority of the votes, the PR can be merged and the candidate will be officially part of their new Committer or Maintainer teams.
+- If a PR remains inactive for 2 days, it is a responsibility of the PR owner to reach out to the reviewers via Discord, bring it up in Community calls or reach out for help in the [hiero-general Discord channel](https://discord.com/channels/905194001349627914/1289954446712770600).
+- If a PR shows no activity for 2 weeks, it will be considered stale and it will be closed with no promotion.
+- A Maintainer or Committer can also nominate a candidate for removal from the teams if the candidate has not shown any activity for 6 months at least or is not responsive.
+- A candidate can also be nominated for removal from the teams if they break code of conduct or has consistently made contributions that are affecting the team.
+- The voting process for removing a candiate is the same as the process for promoting a candidate (via PR).
+
+## Definition of groups
+
+Since Hiero contains multiple projects, we define GitHub **groups** that are used to manage the access to the GitHub repositories.
+The groups are defined in the `config.yaml` file and are used to manage the access to the GitHub repositories that belong to a project.
+Contributors do not have any specific rights, and therefore we do not need to define any groups for them.
+Committers and Maintainers are defined in groups that are used to manage the access to the GitHub repositories.
+For each project we have defined a Committers group and a Maintainers group:
+
+- **`PROJECT-maintainers`**: For each project we have a group that defines the Maintainers of that project.
+- **`PROJECT-committers`**: For each project we have a group that defines the Committers of that project.
+
+There are general groups in the [`config.yaml`](https://github.com/hiero-ledger/governance/blob/main/config.yaml) file as well:
```

Comment:
Are these groups _always_ defined or _optionally_ defined? E.g. `security-maintainers`, `prod-security`, and `sec-ops` seems like it could be overkill for some projects.

Reply:
Those are global and not per project